### PR TITLE
feat: support for linux/arm64 docker images

### PR DESCRIPTION
**Pull request by dkhokhlov**
LOG-11761

### CODE REVIEW

**Comment by dkhokhlov** [comment]
@c-nixon 
failing bc it has depedency on new build image from https://github.com/logdna/build-images/pull/18:

23:44:33  Step 1/10 : ARG ARCH=x86_64
23:44:33  Step 2/10 : FROM docker.io/logdna/build-images:rust-buster-1-stable-$ARCH
23:44:35  manifest for logdna/build-images:rust-buster-1-stable- not found: manifest unknown: manifest unknown

**Comment by dkhokhlov** [comment]
@c-nixon 
build-image done.  now this PR is ready.

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,9 @@
 /bin/target
 **/*.rs.bk
 /.idea
-.cargo_cache/
+.cargo_cache
 **.profraw
 .DS_Store
 *.stamp
+.idea
+/mk*
```
[the remark this answers is not in the document]
Comment (dkhokhlov) [reply .gitignore:11]: something else - I keep mk links in the root.

**Comment by dkhokhlov** [comment]
> I'm a big +1 to checking in scripts that make the dev experience easier but could you also add a short section in the README file to just mention why I would want to use them and maybe a short example of using them?

+1, added & uploaded

**Review by dhable — APPROVED** [review]

**Comment by nodesocket** [comment]
Would love `arm64` support? Just tried installing the agent on my Raspberry Pi Kubernetes cluster and failed with:

```
standard_init_linux.go:228: exec user process caused: exec format error
```

**Comment by dkhokhlov** [comment]
I think arm64 image is not published yet. But it can be built now from current repos on RPi (ARMv8). 
Steps:
1) create build image - will be used to build agent image. 
```
$ uname  -m
aarch64

git clone https://github.com/logdna/build-images.git
cd build-images
./scripts/mk.debian

$ docker images
REPOSITORY                   TAG                                 IMAGE ID       CREATED              SIZE
logdna/build-images          rust-buster-1-stable-aarch64        1acc624d1c98   About a minute ago   4.83GB
logdna/build-images          rust-buster-1-stable-base-aarch64   675e9fe4c3f9   5 minutes ago        1.96GB

```
2) agent image
```
git clone https://github.com/logdna/logdna-agent-v2.git
cd logdna-agent-v2
./scripts/mk.image

$ docker images
REPOSITORY                   TAG                                 IMAGE ID       CREATED          SIZE
logdna-agent-v2              bb188e1-aarch64                     724d2cf6ce8a   9 seconds ago    385MB
logdna/build-images          rust-buster-1-stable-aarch64        1acc624d1c98   7 minutes ago    4.83GB
logdna/build-images          rust-buster-1-stable-base-aarch64   675e9fe4c3f9   11 minutes ago   1.96GB

```

Alternatively you could try my arm64 dev image:
`docker run -it --privileged -e LOGDNA_INGESTION_KEY=$LOGDNA_INGESTION_KEY -v /var/log:/var/log docker.io/dkhokhlov/logdna-agent-dev:3.4.0-beta.1-arm64
`

**Comment by nodesocket** [comment]
@dkhokhlov I appreciate the message and info, but I think I'll just wait until the official containers come out. That way I can continue to use the Helm chart without any modification.

**Pull request merged**